test/framework/alltypes_helpers.go

```diff
@@ -151,7 +151,7 @@ func DumpKubeSystemPodsForCluster(ctx context.Context, input DumpKubeSystemPodsF
 	podList.SetKind("Pod")
 	var listErr error
 	_ = wait.PollUntilContextTimeout(ctx, retryableOperationInterval, retryableOperationTimeout, true, func(ctx context.Context) (bool, error) {
-		if listErr = input.Lister.List(ctx, podList, client.InNamespace(metav1.NamespaceSystem)); listErr != nil {
```

Comment:
@sbueringer @chrischdi WDYT about making a change like this anyway? We could just go through the known CAPI namespaces, but it makes sense to me to just dump all the pods in the workload clusters.

Reply:
Yeah let's just dump all Pods. Feel free to open a separate PR. I just approved the collecting Nodes PR so you can built it on top of it.

Reply:
aggree, dumping all pods would be a great thing esepcially for self-hosted tests :-)